test/cluster_test.go

```diff
@@ -241,6 +239,22 @@ func TestReconnectToNewNodeViaLastMemberList(t *testing.T) {
 	client.Shutdown()
 }
 
+func TestClusterScaleDown(t *testing.T) {
+	cluster, _ = remoteController.CreateCluster("", DefaultServerConfig)
+	member1, _ := remoteController.StartMember(cluster.ID)
+	remoteController.StartMember(cluster.ID)
+	client, _ := hazelcast.NewClient()
+	assert.Len(t, client.Cluster().GetMembers(), 2)
+
+	remoteController.ShutdownMember(cluster.ID, member1.UUID)
+	time.Sleep(3 * time.Second)
```

Comment:
This test is time sensitive.
Instead of sleep, we should either wait the shutdown event via a listener
or we should wait for next assert to be True eventually.